bindings/ros/aditof_roscpp/CMakeLists.txt

```diff
@@ -1,31 +1,34 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(aditof_roscpp)
 
-option(WITH_EXAMPLES "Build examples?" ON)
+option(WITH_ROS_EXAMPLES "Build ROS examples" ON)
 
-## Find catkin macros and libraries
-find_package(catkin REQUIRED COMPONENTS
-  roscpp
-  sensor_msgs
-)
-find_package(aditof REQUIRED)
+# Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS roscpp sensor_msgs)
 
-catkin_package(
-  INCLUDE_DIRS include
-  CATKIN_DEPENDS roscpp sensor_msgs
-)
+# if we create the catkin_ws directory inside the build directory, find_package
+# is not needed
+if(CMAKE_SOURCE_DIR MATCHES ".*aditof_sdk/build.*")
```

Comment:
There is no guarantee that build folder is going to be named 'build'. The cmake should not assume that.

Reply:
Fixed in commit https://github.com/analogdevicesinc/aditof_sdk/pull/241/commits/12add7a2dcb99d8987e9e3b1573f21e1e1957c76